src/pages/surveys/[id]/questions.page.tsx

```diff
@@ -0,0 +1,156 @@
+import omit from 'lodash/omit';
```

Comment:
Note to reviewer: Therefore, Git sees this as a new file, most of the content remains the same. We don't need displayOrder in the URL anymore because all of the questions will be rendered on a single page and using Swiper to navigate through the questions.